backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/payload/TelemetryFixSuggestionResolvedPayload.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarLint Core - Telemetry
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.telemetry.payload;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+import java.util.Map;
+
+public class TelemetryFixSuggestionResolvedPayload {
+
+  @SerializedName("status_by_suggestion_id")
```

Comment:
I am thinking we should name it more distinctively, like `opened_fix_status_by_suggestion_id`, because if at some point we start proposing suggestions inside the IDE directly, the naming will become confusing.

Reply:
I didn't spend too much time thinking about those names but you're right!